scripts/build-sw.mjs

```diff
@@ -0,0 +1,18 @@
+import {generateSW} from 'workbox-build';
+
+generateSW({
+  globDirectory: 'dist/',
+  globPatterns: [
+    '**/*.{css,woff2,png,svg,jpg,js}'
+  ],
+  globIgnores: [
+    'src/**/*'
+  ],
+  runtimeCaching: [
+    {
+      urlPattern: /\.(?:png|jpg|jpeg|svg)$/,
+      handler: 'NetworkFirst',
```

Comment:
https://developer.chrome.com/docs/workbox/reference/workbox-strategies/#type-NetworkFirst

Reply:
nit: we should probably use https://developer.chrome.com/docs/workbox/reference/workbox-strategies/#type-StaleWhileRevalidate